CRM/Core/Page/AJAX/RecurringEntity.php

```diff
@@ -14,7 +14,7 @@ class CRM_Core_Page_AJAX_RecurringEntity {
 
   public static function updateMode() {
     $finalResult = [];
-    if (CRM_Utils_Array::value('mode', $_REQUEST) && CRM_Utils_Array::value('entityId', $_REQUEST) && CRM_Utils_Array::value('entityTable', $_REQUEST)) {
+    if (CRM_Utils_Array::value('mode', $_REQUEST) && CRM_Utils_Array::value('entityId', $_REQUEST) && !empty($_REQUEST['entityTable'])) {
       $mode = CRM_Utils_Type::escape($_REQUEST['mode'], 'Integer');
```

Comment:
Maybe it's just because of the regex used to find these, but is there a reason only the last instance on the line was converted?

Reply:
Applied a slightly different regex to fix this and a couple other spots like it.